modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/repository/SnapshotPersister.java

```diff
@@ -69,33 +77,49 @@
     /** @since 11.1 */
     public static final String PACKAGE_ROOT_NAME = "Packages";
 
-    public static final String Read_Grp = "Everyone";
-
-    public static final String Write_Grp = "members";
+    /** @since 11.1 */
+    public static final String ROOT_TYE_NAME = "Workspace";
 
     public DocumentModel getSubRoot(CoreSession session, DocumentModel root, String name) {
-
         DocumentRef rootRef = new PathRef(root.getPathAsString() + name);
         if (session.exists(rootRef)) {
             return session.getDocument(rootRef);
         }
-        UnrestrictedRootCreator creator = new UnrestrictedRootCreator(session, root.getPathAsString(), name, false);
-        creator.runUnrestricted();
-        // flush caches
-        session.save();
-        return session.getDocument(creator.getRootRef());
+        return createRoot(session, root.getPathAsString(), name, false);
```

Comment:
Why don't we need a privileged session anymore? Maybe we should move `doPrivileged` in `createRoot`?

Reply:
Because the root should be created with the necessary permissions to create subfolders now, i explained this change in the jira ticket.